test/e2e/fleetautoscaler_test.go

```diff
@@ -304,10 +297,13 @@ func TestAutoscalerWebhook(t *testing.T) {
 		// if we could not create the autoscaler, there is no point going further
 		assert.FailNow(t, "Failed creating autoscaler, aborting TestAutoscalerWebhook")
 	}
-	fa := getAllocation(flt)
-	fa, err = alpha1.FleetAllocations(defaultNs).Create(fa)
-	assert.Nil(t, err)
-	assert.Equal(t, stablev1alpha1.GameServerStateAllocated, fa.Status.GameServer.Status.State)
+	gsa := getAllocation(flt)
+	gsa, err = framework.AgonesClient.AllocationV1alpha1().GameServerAllocations(flt.ObjectMeta.Namespace).Create(gsa)
+	if !assert.Nil(t, err) {
+		assert.FailNow(t, "gameserverallocation could not be created")
+	}
+	assert.Equal(t, string(allocationv1alpha1.GameServerAllocationAllocated), string(gsa.Status.State))
```

Comment:
These 6 lines are repeated in multiple places, can you factor it out to a func call

Reply:
Nice. Implementing!